src/hooks/useSections.ts

```diff
@@ -24,14 +30,14 @@ export default function useSections(
     () =>
       activeSections?.filter(
         (config: UserDefinedSection) => config.type === 'autocomplete' || !config.type
```

Comment:
We implicitly consider a section without type to be an autocomplete section. 
[Here's another example](https://github.com/Constructor-io/constructorio-ui-autocomplete/pull/112/files#diff-c72a6a39c2792562569f2b7e56003020f1eefbc541ba6221750d044635f0d1b1R32)

Which let me believe it's worth making it required or always default it to 'autocomplete' if it's not provided instead

Reply:
Where would we add that default?